src/App.tsx

```diff
@@ -1,9 +1,25 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter((movie) => {
+    const { title, description } = movie;
+
+    const normalizeMovieTitle = title.toLowerCase();
+    const normalizeMovieDescription = description.toLowerCase();
+
+    return normalizeMovieDescription.includes(query)
+      || normalizeMovieTitle.includes(query);
+  });
+
+  const handleQueryOnChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
let's give a more explicit name for this handler
```suggestion
  const handleQueryChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```